snow/init.lua

```diff
@@ -1,6 +1,12 @@
+local particlespawner
+
 ctf_match.register_on_new_match(function()
-	minetest.add_particlespawner{
-		amount = 100,
+	if particlespawner ~= nil then
+		minetest.delete_particlespawner(particlespawner)
```

Comment:
How does this part remove the particlespawner when the match is over?

Reply:
See line `8`

Reply:
Oh I misread, I thought it was `particlespawner == nil`, sorry.